tox.ini

```diff
@@ -1,4 +1,6 @@
 [tox]
+skipsdist = True
+usedevelop = True
```

Comment:
I suggest adding to `[testenv]`

```
use_frozen_constraints = true
constrain_package_deps = true
```

to ensure it installs the correct versions of Django/Wagtail

https://tox.wiki/en/4.18.1/config.html#use_frozen_constraints
https://tox.wiki/en/4.18.1/config.html#constrain_package_deps

Reply:
Good shout... Surfaced the issues with 3.8 and 3.9 boto issues...